db/seeds.rb

```diff
@@ -1,7 +1,171 @@
+          "Analysis",
+          "Analytical",
+          "Applications",
+          "Assembling",
+          "Balancing",
+          "Calculating",
+          "Certifications",
+          "Coding",
+          "Compiling Statistics",
+          "Computer",
+          "Computing",
+          "Configuration",
+          "Customer Support",
+          "Data",
+          "Data Mining",
+          "Database Design",
+          "Database Management",
+          "Debugging",
+          "Design",
+          "Development",
+          "Document",
+          "Documentation",
+          "Drilling",
+          "Driving",
+          "Electronics",
+          "Email",
+          "Engineering",
+          "Fabrication",
+          "Finishing",
+          "Follow Specifications",
+          "Grinding",
+          "Hardware",
+          "Implementation",
+          "Information Technology",
+          "Infrastructure",
+          "Languages",
+          "Maintenance",
+          "Manual Dexterity",
+          "Mechanical",
+          "Methodology",
+          "Modeling",
+          "Modification",
+          "Needs Analysis",
+          "Network Architecture",
+          "Network Security",
+          "Networking",
+          "New Technologies",
+          "Operating Systems",
+          "Operations",
+          "Performance",
+          "Planning",
+          "Platforms",
+          "Photography",
+          "Problem Solving",
+          "Product Reliability",
+          "Proficiency",
+          "Project Management",
+          "Programming",
+          "Quality Assurance",
+          "Quality Control",
+          "Quantitative Research",
+          "Quantitative Reports",
+          "Refinishing",
+          "Remodeling",
+          "Repairing",
+          "Reporting",
+          "Restoration",
+          "Security",
+          "Servers",
+          "Storage",
+          "Software",
+          "Solution Delivery",
+          "Sorting",
+          "Statistical Analysis",
+          "Structures",
+          "Systems Analysis",
+          "Technical Support",
+          "Technical Writing",
+          "Technology",
+          "Testing",
+          "Tools",
+          "Training",
+          "Troubleshooting",
+          "Usability"]
+
+
+INTERESTS = ["Sports such as football, baseball, golf, soccer, tennis etc.",
+            "Photography/Videography",
+            "Movies and Theater",
+            "Video games",
+            "Traveling and Exploring different places",
+            "Reading",
+            "Listening to music",
+            "Playing a musical instrument",
+            "Dancing",
+            "Acting",
+            "Surfing the Internet/Blogging/Programming",
+            "Gardening",
+            "Collecting",
+            "Bowling",
+            "Iceskating",
+            "Taking care of pets",
+            "Trekking",
+            "Fitness",
+            "Extreme sports and activities (such as sky diving, mountain climbing etc.)",
```

Comment:
Line is too long. [89/80]<br>Align the elements of an array literal if they span more than one line.